client-metrics-reporter/src/main/java/io/strimzi/kafka/metrics/prometheus/common/AbstractReporter.java

```diff
@@ -27,6 +27,11 @@ public abstract class AbstractReporter {
     // allowlist is updated we can update the matching metrics.
     private final Map<Object, MetricWrapper> disallowedMetrics = new ConcurrentHashMap<>();
 
+    /**
+     * Empty constructor
+     */
+    public AbstractReporter() { }
```

Comment:
Do we need to expose it? Can't we just use private or protected?
```suggestion
    public AbstractReporter() { }
```

Reply:
Yes it needs to be `public`, we have concrete implementations in other packages.